4.1/README.md

```diff
@@ -0,0 +1,218 @@
+# Nominatim Docker (Nominatim version 4.1)
+
+## Table of contents
+
+  - [Automatic import](#automatic-import)
+  - [Configuration](#configuration)
+    - [General Parameters](#general-parameters)
+    - [PostgreSQL Tuning](#postgresql-tuning)
+    - [Import Style](#import-style)
+    - [Flatnode files](#flatnode-files)
+  - [Persistent container data](#persistent-container-data)
+  - [OpenStreetMap Data Extracts](#openstreetmap-data-extracts)
+  - [Updating the database](#updating-the-database)
+  - [Custom PBF Files](#custom-pbf-files)
+  - [Importance Dumps, Postcode Data, and Tiger Addresses](#importance-dumps-postcode-data-and-tiger-addresses)
+  - [Development](#development)
+  - [Docker Compose](#docker-compose)
+  - [Assorted use cases documented in issues](#assorted-use-cases-documented-in-issues)
+
+***
+
+## Automatic import
+
+Download the required data, initialize the database and start nominatim in one go
+
+```
+docker run -it --rm \
```

Comment:
I wonder if we should keep the `--rm` in the instructions. We keep getting complaints from people who don't know what it means.

WDYT?

Reply:
Good point, I always tend to forget that too many people just copy & paste our instructions without knowing what they do :)